docs/tutorials/scheduling.rst

```diff
@@ -0,0 +1,269 @@
+can also give a default duration if you aren't able to give one for
+each pair of configurations.
+
+.. code-block:: python
+
+    >>> from astroplan.scheduling import Transitioner
+
+    >>> transitioner = Transitioner(.8*u.deg/u.second,
+    ...                             {'filter':{('B','G'): 10*u.second,
+    ...                                        ('G','R'): 10*u.second,
+    ...                                        'default': 30*u.second}})
+
+The transitioner now knows that it takes 10 seconds to go from 'B' to 'G',
+or from 'G' to 'R' but has to use the default transition time of 30 seconds
+for any other transition between filters. Non-transitions, like 'g' to 'g',
+will not take any time though.
+
+.. _scheduling-scheduling:
+
+Scheduling
+==========
+
+Now all we have left is to initialize the scheduler, and run it on our
+list of blocks. There are currently two schedulers to chose from in
+astroplan.
+
+The first is a sequential scheduler. It starts at the start_time and
+scores each block (constraints and target) at that time and then
+schedules it, it then moves to where the first observing block stops
+and repeats the scoring and scheduling on the remaining blocks.
+
+.. code-block:: python
+
+    >>> from astroplan.scheduling import SequentialScheduler
+    >>> from astroplan.scheduling import Schedule
+
+    >>> seq_scheduler = SequentialScheduler(constraints = global_constraints,
+    ...                                     observer = apo,
+    ...                                     transitioner = transitioner)
+    >>> schedule = Schedule(start_time, end_time)
+
+    >>> sequential_schedule = seq_scheduler(blocks, schedule)
+
+The second is a priority scheduler. It sorts the blocks by their
+priority (multiple blocks with the same priority will stay in the
+order they were in), then schedules them one-by-one at the best
+time for that block (highest score).
+
+.. code-block:: python
+
+    >>> from astroplan.scheduling import PriorityScheduler
+
+    >>> prior_scheduler = PriorityScheduler(constraints = global_constraints,
+    ...                                     observer = apo,
+    ...                                     transitioner = transitioner)
+    >>> schedule = Schedule(start_time, end_time)
+
+    >>> priority_schedule = prior_scheduler(blocks, schedule)
+
+Now that you have a schedule there are a few ways of viewing it.
+One way is to have it print a table where you can show, or hide,
+unused time and transitions with ``show_transitions`` and
+``show_unused`` (default is showing transitions and not unused).
+
+.. code-block:: python
+
+    >>> sequential_schedule.to_table()
+
+The other way is to plot the schedule against the airmass of the
+targets.
+
+.. code-block:: python
+
+    >>> from astroplan.plots import plot_schedule_airmass
+    >>> import matplotlib.pyplot as plt
+
+    >>> plt.figure(figsize = (14,6))
+    >>> plot_schedule_airmass(priority_schedule)
+    >>> plt.legend(loc = 1)
```

Comment:
Would you use the location key `"upper right"` rather than `1`? It's a bit more descriptive.